pkg/plugins/manager/loader/assetpath/assetpath.go

```diff
@@ -25,23 +25,23 @@ func ProvideService(cdn *pluginscdn.Service) *Service {
 // Base returns the base path for the specified plugin.
 func (s *Service) Base(pluginJSON plugins.JSONData, class plugins.Class, pluginDir string) (string, error) {
 	if class == plugins.ClassCore {
-		return path.Join("public/app/plugins", string(pluginJSON.Type), filepath.Base(pluginDir)), nil
+		return path.Join("/public/app/plugins", string(pluginJSON.Type), filepath.Base(pluginDir)), nil
```

Comment:
I realize that the sandbox code is currently not handling the case of grafana running inside a sub path correctly (custom.ini -> server -> serve_from_sub_path: true) 

The backend is not returning the complete path to the plugin files complete and I can't figure how old system js included that sub path on its code. My best guess is that the lack of this leading slash made it work.

Nevertheless I see in this PR you are adding logic to handle the case and the sandbox will have to add that logic too. Could we instead return the correct path directly from the backend instead of letting frontend decide how to handle?